docs/pages/7_0_0_release_notes.md

```diff
@@ -399,6 +401,23 @@ The metrics framework has been made simpler and more general.
 * Rule tests, that use {% jdoc test::testframework.SimpleAggregatorTst %} or {% jdoc test::testframework.PmdRuleTst %} work as before without change, but use
   now JUnit5 under the hood. If you added additional JUnit4 tests to your rule test classes, then you'll need to upgrade them to use JUnit5.
 
+#### Language Modules
```

Comment:
Not sure, how detailed we should mention this change in the release notes...

Reply:
We can probably remove some detailsl. I don't think many people care about that change, the details are better kept in the doc pages that you updated

Reply:
I've added a TODO on #4373 to review this section.